source/en/skins-system.rst

```diff
@@ -0,0 +1,163 @@
+Skins system
+------------
+
+On this page you'll find information about available endpoints of Ely.by's service of skins system. You can use any of
+them as an secondary or primary source of skins for your project.
+
+Ely.by's service of skins system provides `proxying of textures from Minecraft premium users <#textures-proxy>`_,
+which means that when using this service players will see both premium Minecraft users' skins and Ely.by users' skins.
```

Comment:
```suggestion
which means that using this service, your players will see both premium Minecraft users' skins and Ely.by users' skins.
```
Maybe you want to add "your players", if you are trying to market this service to servers holders.